src/EnergyPlus/GroundTemperatureModeling/FiniteDifferenceGroundTemperatureModel.cc

```diff
@@ -414,7 +394,7 @@ void FiniteDiffGroundTempsModel::developMesh()
             // Constant thickness mesh here
             thisCell.thickness = surfaceLayerCellThickness;
 
-        } else if (i > surfaceLayerNumCells && i <= (centerLayerNumCells + surfaceLayerNumCells)) {
+        } else if (i <= (centerLayerNumCells + surfaceLayerNumCells)) {
```

Comment:
[src/EnergyPlus/GroundTemperatureModeling/FiniteDifferenceGroundTemperatureModel.cc:417]:(style),[knownConditionTrueFalse],Condition 'i>surfaceLayerNumCells' is always true